_posts/2018-11-30-grpc-stacks.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: post
+title: gRPC Language Stacks
+published: false
+permalink: blog/grpc-stacks
+author: Carl Mastrangelo
+author-link: https://carlmastrangelo.com
+company: Google
+company-link: https://www.google.com
+---
+
+Here is a high level overview of the gRPC Stacks.  Each of the **10** default languages supported by gRPC has multiple layers, allowing you to customize what pieces you want in your application.
+
+<!--more-->
+
+
+There are three main stacks in gRPC: C-core, Go, and Java.  Most of the languages are thin wrappers on top of the [C-based](https://github.com/grpc/grpc/tree/v1.16.1/src/core) gRPC core library:
```

Comment:
Link to master branch as 1.16.1 will be old soon.

Reply:
I had picked a tag because links tend to break over time (re-orging code).  Done, though.